src/main/java/Task.java

```diff
@@ -0,0 +1,41 @@
+public class Task {
+    private String description;
+    private boolean isDone;
+    private int taskNumber;
+    private static int numberOfTasks = 0;
```

Comment:
Just curious, is there a reason why you would use a static variable here instead of the main function?